Sandbox/CustomerExamples/UpdatingTheLegend/src/index.ts

```diff
@@ -56,7 +57,40 @@ async function initSciChart() {
 
     sciChartSurface.renderableSeries.add(lineSeries1, lineSeries2);
     // To place the Legend outside of the canvas uncomment this line
```

Comment:
Change this comment to 
// Pass placementDivId to place the legend outside the chart

Reply:
done